src/main/resources/db/migration/changes/007-create-Instructors.json

```
@@ -0,0 +1,43 @@
+{ "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "Instructor-1",
```

Can you make the id be the same as the filename?

I realize this was not clear; we're just now starting to follow this convention.

```suggestion
        "id": "007-create-Instructors.json",
```